stylesheets/_component.block-list.scss

```diff
@@ -101,6 +101,10 @@ $width: 500px;
     }
 }
 
+.block-list__item.is-disabled {
```

Comment:
It's a good question which warrants explaining :) in Pulsar we use `is-disabled` as a common modifier class that is supported across many components and helpers. 

Having `block-list__item--disabled` puts a lot of burden on the developer wanting to quickly put together a Pulsar UI, needing to know that they're using a block list, that the thing is a block list item, and that we're using BEM to internally organise styles. It'd need documenting, and almost always have to be looked up when required.

Far, far easier that someone implementing a Pulsar UI should get predictable results by adding `is-disabled` to anything, whether it's a link, a button, or a more complex component like a form control.

Similar modifiers would be `is-active` `is-inactive` `is-open` `is-closed` etc.

Bit more info here: https://jadu.github.io/pulsar/styleguides/state/